### PR TITLE
hyejoo bot uses sushii image server

### PR DESCRIPTION
**Pull request by chuuloona**
is it okay without the github link? ididn't upload mine on it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,7 @@ SUSHII_IMG_QUALITY=70
 
 * [sushii bot]
 * [haseul bot]
+* [hyejoo bot]
```
Comment (drklee3) [inline README.md:201]: If you don't have it on GitHub that's fine! Does it have a website or anything? If not, you can remove the brackets.

```suggestion
* [hyejoo bot]
* hyejoo bot
```

Comment (chuuloona) [reply README.md:201]: can it be a top gg link? if not issok!

Comment (drklee3) [reply README.md:201]: Yeah that's fine, you can add the link at the bottom with the other links.

Comment (chuuloona) [reply README.md:201]: [hyejoo bot](https://top.gg/bot/733035786664149104)

is this fine? sorry, im new to pr'ing.

File: README.md [hunk 2]
```diff
@@ -198,6 +198,8 @@ SUSHII_IMG_QUALITY=70
...
+* hyejoo bot
```
Comment (drklee3) [inline README.md]: Oops my earlier suggestion had a double, just remove the extra one,

```suggestion
```

then add this at the bottom with the other links and that should be good!

```md
[hyejoo bot]: https://top.gg/bot/733035786664149104
```

**Review by drklee3 — COMMENTED** [review]
Yeah it's fine, just so that the link is at the bottom.

File: README.md (hunk 1 above)
Comment (chuuloona) [reply README.md:201]: [hyejoo bot]: https://top.gg/bot/733035786664149104

**Review by chuuloona — COMMENTED** [review]
[hyejoo bot]: https://top.gg/bot/733035786664149104

File: README.md (hunk 1 above)
Comment (drklee3) [reply README.md:201]: It should go in the file, this is the comment so it won't modify the file.

File: README.md [hunk 3]
```diff
@@ -198,6 +198,7 @@ SUSHII_IMG_QUALITY=70
 
 * [sushii bot]
 * [haseul bot]
+* [hyejoo bot]
 * Use sushii-image-server? Feel free to open a PR to add to the list!
 
 [PM2]: https://github.com/Unitech/pm2
```
Comment (drklee3) [inline README.md:204]: GitHub isn't letting me give a suggestion at the very bottom, but should be easier to just add with this 😄 

```suggestion
[PM2]: https://github.com/Unitech/pm2
[hyejoo bot]: https://top.gg/bot/733035786664149104
```

**Review by drklee3 — COMMENTED** [review]
Just the link at the bottom now

**Review by drklee3 — APPROVED** [review]
Great! Thanks for adding to the list.

**Pull request merged**